xdsl/dialects/riscv.py

```diff
@@ -1531,23 +1531,13 @@ class FMVOp(RdRsOperation[FloatRegisterType, FloatRegisterType]):
     """
     A pseudo instruction to copy contents of one float register to another.
 
-    Equivalent to `fsw rs, 0(sp); flw rd, 0(sp)`.
-
-    This is a much more expensive operation than mv, and does not correspond to an
-    instruction in the RISC-V F extension. Possibly due to this, there is no common
-    assembly pseudo-op for it, so it must be rewritten before assembly printing.
+    Equivalent to `fsw rs, 0(sp); flw rd, 0(sp)` or `fsgnj.s rd, rs, rs`
```

Comment:
I'm not sure about the `fsw rs, 0(sp); flw rd, 0(sp)` part especially with the word *equivalent*. `fsgnj.s rd, rs, rs` is enough?

Reply:
Yeah I'll rephrase this.